JamVM doesn't support the '-ea' flag

--- origsrc/langtools-jdk6-b20/src/share/bin/launcher.sh-template	2010-06-11 12:23:59.000000000 -0500
+++ src/langtools-jdk6-b20/src/share/bin/launcher.sh-template	2010-10-21 15:39:50.624165200 -0500
@@ -45,7 +45,7 @@ if [ "$LANGTOOLS_USE_BOOTCLASSPATH" != "
 fi
 
 # tools currently assumes that assertions are enabled in the launcher
-ea=-ea:com.sun.tools...
+#ea=-ea:com.sun.tools...
 
 # Any parameters starting with -J are passed to the JVM.
 # All other parameters become parameters of #PROGRAM#.
@@ -67,4 +67,4 @@ done
 unset DUALCASE
 
 IFS=$nl
-"#TARGET_JAVA#" "${bcp:+-Xbootclasspath/p:"$bcp"}" ${ea} ${javaOpts} -jar "${mydir}"/../lib/#PROGRAM#.jar ${toolOpts}
+"#TARGET_JAVA#" "${bcp:+-Xbootclasspath/p:"$bcp"}" ${ea} ${javaOpts} -jar "${mylib}"/#PROGRAM#.jar ${toolOpts}
